ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -122,8 +112,6 @@ class ( SL.ShelleyBasedEra era
 
       , SL.AdditionalGenesisConfig era ~ ()
```

Comment:
The comment about `AdditionalGenesisConfig` being temporarily equivalent to `()` was removed but we've never established with ledger-spec team if this type family is ever going to be used (it seemed that `TranslationContext` is being used in Alonzo). It `AdditionalGenesisConfig` is some sort of "future-proof" mechanism, it would make sense to document it here.

Reply:
Nice catch. I created a follow-up Issue.

https://github.com/input-output-hk/ouroboros-network/issues/3143